Content.Server/Guardian/GuardianSystem.cs

```diff
@@ -240,6 +240,8 @@ private void OnGuardianDamaged(EntityUid uid, GuardianComponent component, Damag
         {
             if (args.DamageDelta == null) return;
 
+
+
```

Comment:
Huh?
```suggestion
```

Reply:
Oh oops this was from when I was trying something else